rpcs3/Emu/Io/usb_device.cpp

```diff
@@ -200,24 +200,20 @@ bool usb_device_emulated::open_device()
 
 u32 usb_device_emulated::get_descriptor(u8 type, u8 index, u8* buf, u32 buf_size)
 {
-	u32 expected_count = 2;
+	std::array<u8, 2> header; header = {header.size(), type}; // Reference: https://github.com/RPCS3/rpcs3/pull/14903#discussion_r1429197999
```

Comment:
It's done automatically via git history, that's I suggested earlier to not change many lines which are not part of the pr.

Reply:
Also put a newline inbetween

Reply:
Done. Thank you.